src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -48,20 +48,20 @@
    * Convenience method.
    */
   public static UnitAttachment get(final UnitType type) {
-    final UnitAttachment rVal = (UnitAttachment) type.getAttachment(Constants.UNIT_ATTACHMENT_NAME);
-    if (rVal == null) {
+    final UnitAttachment unitAttachment = (UnitAttachment) type.getAttachment(Constants.UNIT_ATTACHMENT_NAME);
+    if (unitAttachment == null) {
       throw new IllegalStateException("No unit type attachment for:" + type.getName());
     }
-    return rVal;
+    return unitAttachment;
```

Comment:
This method should just call the other getMethod and return that value or something similar.